doc/api/net.md

```diff
@@ -625,15 +625,16 @@ initialDelay will leave the value unchanged from the default
 
 Returns `socket`.
 
-### socket.setNoDelay([noDelay])
+### socket.setNoDelay([enable])
```

Comment:
or maybe when "enable" is false, nagle is disabled... so no dealy is true? In which case, the meaning of true/false to this function would be inverted, so that's an API change, and we would never do it.